lib/_http_outgoing.js

```diff
@@ -57,6 +58,7 @@ const kIsCorked = Symbol('isCorked');
 
 const RE_CONN_CLOSE = /(?:^|\W)close(?:$|\W)/i;
 const RE_TE_CHUNKED = common.chunkExpression;
+const HIGH_WATER_MARK = getDefaultHighWaterMark();
```

Comment:
Now that c065773cc56 has landed, this needs to be removed?

Reply:
Fixed